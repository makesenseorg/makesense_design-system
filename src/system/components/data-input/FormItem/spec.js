import { shallowMount } from '@vue/test-utils'
import Comp from './FormItem.vue'

describe('FormItem.vue', () => {
  it('matches snapshot', () => {
    const wrapper = shallowMount(Comp, {
      provide: {
        $parentInput () {
          return {}
        }
      }
    })
    expect(wrapper.element).toMatchSnapshot()
  })
})