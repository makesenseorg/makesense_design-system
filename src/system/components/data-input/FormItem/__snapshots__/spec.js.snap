// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormItem.vue matches snapshot 1`] = `
<div
  class="ds-form-item"
>
  <ds-input-label-stub />
    
  <ds-input-error-stub />
</div>
`;
